optaplanner-examples/src/main/java/org/optaplanner/examples/coachshuttlegathering/domain/location/RoadLocationArc.java

```diff
@@ -67,4 +67,12 @@ public void setShuttleDuration(int shuttleDuration) {
     // Complex methods
     // ************************************************************************
 
+    @Override
+    public String toString() {
+        return "Road arc [coach: "
```

Comment:
For example? I checked some domain classes and they usually return name but RoadLocationArc has no name. I couldn't find a pattern in the rest of domain classes.